confdgnmi/src/confd_gnmi_adapter.py

```diff
@@ -58,126 +58,238 @@ def set(self, prefix, path, val):
         """
         pass
 
-
     class SubscriptionHandler(ABC):
```

Comment:
Why is this large class nested, is it useful? It definitely makes reading the code more difficult; the same holds for its descendants/implementations.

Reply:
I was thinking about it. I believe it belongs there as it only groups functionality that is handled by adapter (subscription), not used anywhere else.

Reply:
It should definitely stay within the same module (which consists of only those two classes), but I don't see any reason why make reading the code more difficult.

Reply:
WONT FIX for now